src/CoreWCF.Primitives/src/CoreWCF/Configuration/ConfigurationStrings.cs

```diff
@@ -0,0 +1,551 @@
+
+namespace CoreWCF.Configuration
+{
+    static class ConfigurationStrings
+    {
```

Comment:
This entire class shouldn't need to exist as it should only be used for configuration.